### PR TITLE
Update course tests and standardize some messages

### PR DESCRIPTION
**Pull request by teriaiw**

### CODE REVIEW

File: src/test/java/igrad/logic/commands/course/CourseAchieveCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        assertFalse(courseAchieveCommand.equals(module));
+
+        //different course achieve command
+        CourseAchieveCommand otherCourseAchieveCommand;
+        Optional<Cap> otherCapToAchieve = Optional.of(new Cap("0"));
+
+        otherCourseAchieveCommand = new CourseAchieveCommand(otherCapToAchieve);
+
+        //For some weird reason the below returns true. Am not sure of it so am commenting it out first. (Teri)
+        //assertFalse(courseAchieveCommand.equals(otherCourseAchieveCommand));
```
Comment (nathanaelseen) [inline src/test/java/igrad/logic/commands/course/CourseAchieveCommandTest.java]: ok

Comment (nathanaelseen) [reply src/test/java/igrad/logic/commands/course/CourseAchieveCommandTest.java]: you need to check `CourseAchieveCommand` whether you  have implemented `equals` correctly

File: src/test/java/igrad/model/course/CourseInfoTest.java [hunk 2]
```diff
@@ -394,6 +395,26 @@ public void computeCap_twoModulesSuGrade_returnsCapZero() {
         assertEquals(expectedCap, computedCap);
     }
 
+    //@@author teriaiw
```
Comment (nathanaelseen) [inline src/test/java/igrad/model/course/CourseInfoTest.java:398]: good

File: src/test/java/igrad/model/course/CourseInfoTest.java [hunk 3]
```diff
@@ -410,28 +431,45 @@ public void equals() {
         Module module = new ModuleBuilder().build();
         assertFalse(BCOMPSCI.equals(module));
 
+        assertFalse(BCOMPSCI.equals(GENERAL_ELECTIVES));
```
Comment (nathanaelseen) [inline src/test/java/igrad/model/course/CourseInfoTest.java:434]: redundant

File: src/test/java/igrad/model/course/CourseInfoTest.java [hunk 4]
```diff
@@ -410,28 +431,45 @@ public void equals() {
...
+
         CourseInfo otherCourseInfo;
 
         // different course info; only cap different
         otherCourseInfo = new CourseInfoBuilder(BCOMPSCI)
             .withCap(VALID_COURSE_CAP_BCOMPSEC)
             .build();
-        assertFalse(GENERAL_ELECTIVES.equals(otherCourseInfo));
+        assertFalse(BCOMPSCI.equals(otherCourseInfo));
```
Comment (nathanaelseen) [inline src/test/java/igrad/model/course/CourseInfoTest.java:442]: oh  oops, thanks Teri

File: src/test/java/igrad/model/course/CourseInfoTest.java [hunk 5]
```diff
@@ -410,28 +431,45 @@ public void equals() {
...
 
         // different course info; only credits different
         otherCourseInfo = new CourseInfoBuilder(BCOMPSCI)
             .withCredits(VALID_COURSE_CREDITS_REQUIRED_BCOMPSEC,
                 VALID_COURSE_CREDITS_FULFILLED_BCOMPSEC)
             .build();
-        assertFalse(GENERAL_ELECTIVES.equals(otherCourseInfo));
+        assertFalse(BCOMPSCI.equals(otherCourseInfo));
```
Comment (nathanaelseen) [inline src/test/java/igrad/model/course/CourseInfoTest.java:449]: thanks

File: src/test/java/igrad/model/course/CourseInfoTest.java [hunk 6]
```diff
@@ -410,28 +431,45 @@ public void equals() {
...
         CourseInfo other = new CourseInfoBuilder(BCOMPSCI)
             .withCap(VALID_COURSE_CAP_BCOMPSEC)
             .withCredits(VALID_COURSE_CREDITS_REQUIRED_BCOMPSEC,
                 VALID_COURSE_CREDITS_FULFILLED_BCOMPSEC)
             .build();
-        assertFalse(GENERAL_ELECTIVES.equals(other));
+        assertFalse(BCOMPSCI.equals(other));
+
+        //different course info; cap, credits, semesters, all different
+        other = new CourseInfoBuilder(BCOMPSCI)
```
Comment (nathanaelseen) [inline src/test/java/igrad/model/course/CourseInfoTest.java:466]: good but you could have added another case  above, where only the `semesters` was different

Comment (teriaiw) [reply src/test/java/igrad/model/course/CourseInfoTest.java:466]: I did, its slightly above

**Pull request merged**